src/aktualizr_lite/main.cc

```diff
@@ -189,7 +189,7 @@ int main(int argc, char *argv[]) {
 
   bpo::variables_map commandline_map = parse_options(argc, argv);
 
-  int r = -1;
+  int r = 0;
```

Comment:
This is fine by me. However, the logic was copied from:
 https://github.com/advancedtelematic/aktualizr/blob/master/src/aktualizr_primary/main.cc#L105

You might want to fix that in the other binaries as well so that things stay consistent (in code and for user error codes)

Reply:
Yes, I agree, thx for pointing to it.